lib/run_loop/device_agent/client.rb

```diff
@@ -574,6 +574,45 @@ def springboard_alert_visible?
       end
 
       # @!visibility private
+      def dismiss_springboard_alert(button_title)
+        parameters = { :button_title => button_title }
+        request = request("dismiss-springboard-alert", parameters)
+        client = http_client(http_options)
+        response = client.post(request)
+        hash = expect_300_response(response)
+
+        if hash["error"]
+          raise RuntimeError, %Q[
+Could not dismiss SpringBoard alert by touching button with title '#{button_title}':
+
+#{hash["error"]}
+
+]
+        end
+        true
+      end
+
+      # @!visibility private
+      def dismiss_springboard_alerts_automatically
```

Comment:
> set_

You are right, this is better - particularly at this API level.

> !

Also correct.

When I implement the public API (in Calabash iOS), I want to use:

```
dismiss_springboard_alerts_automatically! 
dismiss_springboard_alerts_manually!
```

for clarity - there is absolutely no ambiguity.

On the other hand, maybe it would be best to use the `set_` idiom for discoverability.

I am trying to avoid using the word `dismissal` in the API and in the documentation because I have a feeling (based on Gitter chats) that it confuses non-native English speakers.


Thanks.